backend/src/db/schemas/secrets.ts

```diff
@@ -23,7 +23,7 @@ export const SecretsSchema = z.object({
   secretCommentTag: z.string().nullable().optional(),
   secretReminderNote: z.string().nullable().optional(),
   secretReminderRepeatDays: z.number().nullable().optional(),
-  skipMultilineEncoding: z.boolean().default(false).nullable().optional(),
+  skipMultilineEncoding: z.boolean().default(false).optional(),
```

Comment:
The migration file has this as a bool not nullable so not sure why it was set to nullable here